src/Components/ClusterRules/ClusterRules.cy.js

```diff
@@ -251,6 +251,17 @@ describe('cluster rules table', () => {
         .should('have.length', RULES_ENABLED);
     });
 
+    it('will reset filters but not pagination and sorting', () => {
+      filterApply({ description: 'Lo' });
+
+      cy.get('th[data-label="Description"]').find('button').click();
+      cy.get(TOOLBAR).find('button').contains('Reset filters').click();
+      cy.get(TOOLBAR).find(CHIP_GROUP).should('not.exist');
+      cy.get('th[data-label="Description"]')
```

Comment:
you also can use `checkPaginationSelected` here to check the pagination (=can we check for pagination and sorting in tests for all pages?)

Reply:
ClusterRules doesn't have pagination.